CHANGELOG.md

```diff
@@ -17,6 +19,8 @@ This project (not yet) adheres to [Semantic Versioning](https://semver.org/spec/
 ### Changed
 
 - make wording for ai report toggle more specific
+- added new modifier classes to `PollResults.jsx`, `PollOpenQuestion.jsx` and `PollQuestion.jsx` for customized paddings
+- replaced `<span></span>` with react fragment to keep consistency
```

Comment:
@hom3mad3 @goapunk dunno why this changelog wasn't there when I created the 2406.3 release, or I totally missed it, though i used the parser script.